frontend/assets/stylesheets/utilities/_util-visibility.scss

```diff
@@ -62,6 +62,7 @@
     }
 }
 
+
```

Comment:
Minor nit - try to keep pull requests smaller by leaving out unnecessary changes - makes it easier to focus on the important changes.